src/store/ducks/plans.ts

```diff
@@ -389,53 +388,62 @@ export interface PlanFilters {
   parentJurisdictionId?: string /** jurisdiction parent id */;
   reason?: FIReasonType /** plan FI reason */;
   statusList?: string[] /** array of plan statuses */;
+  title?: string /** plan title */;
 }
 
 /** plansArrayBaseSelector select an array of all plans
  * @param state - the redux store
  */
-export const plansArrayBaseSelector = (planKey?: string) => (state: Registry): Plan[] =>
+export const plansArrayBaseSelector = (planKey?: string) => (state: Partial<Store>): Plan[] =>
```

Comment:
@kelvin-muchiri what led to use changing from `Registry` to `Partial<Store>` in this file?

Reply:
@moshthepitt  `state` from `mapStateToProps` is not of type `Registry` and will raise lint warning if passed as an argument. Or which is the best way to solve this?